docs/WritingTests.asciidoc

```diff
@@ -1408,3 +1408,13 @@ state of the specified git refspec. For example the asset caching
 functionality will override the +PRODUCTDIR+ variable. The above method can
 still be used in this case in the common case that the test schedule does not
 need to be changed or in case a custom schedule is defined by +SCHEDULE+.
+
+== Controling list of log files to upload
+
+During every job run the worker produces various log files. By specifying the variable +AUTOINST_UPLOAD_FROM_POOL+
+with a comma separated list of filenames it is possible to control which files should be uploaded to job results.
+Default set of files is defined in https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Job.pm
+( +UPLOAD_FROM_POOL+ constant ).
+Keep in mind that definition of +AUTOINST_UPLOAD_FROM_POOL+ variable will overwrite default list. So in case you want
+just add variable to list you need to specify your file plus all default. To see all files available to upload examine
+/var/lib/openqa/pool/<worker_id> during test execution.
```

Comment:
How about you sprinkle in some articles? ;)

Reply:
Or add a hint to read the this paragraph with Russian accent :-)

Reply:
@Martchus "Spasiba" comrad :P

Reply:
@Martchus good idea. I already read out the paragraph to @asmorodskyi with Russian accent to prove my point ;)

Reply:
please check latest version ...